### PR TITLE
Fixed bugs in case finding report

### PR DESCRIPTION
**Pull request by kkrampa**
@czue

### CODE REVIEW

**Review by czue — APPROVED** [review]
code looks fine, but can you explain how this works (maybe with some comments in the code)? I think you added a hack to return all descendant location IDs in the `value` which makes sense, but i don't totally understand what's going on in `selected`. also, would be nice if this could be generalized more, but we can think about that separately

File: custom/enikshay/reports/filters.py [hunk 1]
```diff
@@ -25,12 +25,29 @@ def options(self):
 
     @property
     def selected(self):
```
Comment (kkrampa) [inline custom/enikshay/reports/filters.py]: When a user selects a non-leaf location and he/she hits apply button descendants of selected location will also appear in a select box. That's why I overrode that method to not return descendants. Does it make sense to you?

Comment (czue) [inline custom/enikshay/reports/filters.py]: yep, it does, thanks

**Comment by kkrampa** [comment]
@czue :white_check_mark: 
Can we move this forward?

**Comment by czue** [comment]
sorry was just waiting on build.

**Pull request merged**